pkg/api/extensions/accessor_test.go

```diff
@@ -26,6 +24,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	"time"
```

Comment:
why is the import moved down? generally, we try to have 3 blocks, first block for stdlib imports, second block for repository imports, and third block for third-party imports.

Reply:
I should use goimports to be consistent